src/AccountLabel.jsx

```diff
@@ -1,52 +1,55 @@
 const React = require('react');
-const {bonds, toChecksumAddress} = require('oo7-parity');
-const {ReactiveComponent, Rimg} = require('oo7-react');
-const {isNullData} = require('oo7-parity');
-const {Label, Icon} = require('semantic-ui-react');
-const {AccountIcon} = require('./AccountIcon');
+const Api = require('@parity/api');
+const { bonds, toChecksumAddress, isNullData } = require('oo7-parity');
+const { ReactiveComponent, Rimg } = require('oo7-react');
+const { Label, Icon } = require('semantic-ui-react');
+const { AccountIcon } = require('./AccountIcon');
 
 class AccountLabel extends ReactiveComponent {
 	constructor () {
 		super(['address']);
 	}
 	render () {
 		if (this.state.address === null) {
-			return (<Label
-				as='a'
-				basic={false}
-				image
-				color='red'
-				basic={true}
-				style={{
-					borderTopLeftRadius: '16px',
-					borderBottomLeftRadius: '16px'
-				}}
-			>
-				<Icon name='warning circle' style={{height: '100%'}} />
-				undefined
-			</Label>);
+			return (
+				<Label
+					as='a'
+					basic={ false }
+					image
+					color='red'
+					style={ {
+						borderTopLeftRadius: '16px',
+						borderBottomLeftRadius: '16px'
+					} }
+				>
+					<Icon name='warning circle' style={ { height: '100%' } } />
+					undefined
+				</Label>);
 		} else if (isNullData(this.state.address)) {
-			return (<Label
-				as='a'
-				color={'black'}
-				image
-				basic={true}
-				style={{
-					borderTopLeftRadius: '16px',
-					borderBottomLeftRadius: '16px'
-				}}
-			>
-				<Icon name='ban' style={{height: '100%'}} />
-				Null
-			</Label>);
+			return (
+				<Label
+					as='a'
+					color={ 'black' }
+					image
+					basic
+					style={ {
+						borderTopLeftRadius: '16px',
+						borderBottomLeftRadius: '16px'
+					} }
+				>
+					<Icon name='ban' style={ { height: '100%' } } />
+					Null
+				</Label>);
 		} else {
-			let a = toChecksumAddress(this.state.address);
-			return (<AccountLabelAux
-				address={a}
-				names={bonds.namesOf(a)}
-				badges={bonds.badgesOf(a)}
-				noicon={this.props.noicon}
-			/>);
+			let a = Api.util.toChecksumAddress(this.state.address);
```

Comment:
We are importing the `toChecksumAddress` above already - doing this we adding another dependency?